### PR TITLE
[Documentation] Mention the extra file checks in the pipeline flow description

### PR DESCRIPTION
**Pull request by cmadjar**
Based on @MounaSafiHarab's feedback from PR #219, modified the pipeline flow section to mention the extra file checks performed when determining the MRI protocol.

### CODE REVIEW

File: docs/02-Install.md [hunk 1]
```diff
@@ -93,13 +93,17 @@ Typically, images insertion into LORIS is performed via the following steps:
...
+         **Imaging Browser** module and can be displayed in 3D using 
+         **BrainBrowser**.
 
     b. If a scan does not match any of the protocol defined in the
-         `mri_protocol` table, then the MINC image of this scan will be stored
-         into the `mri_protocol_violated_scans` table. This violated image is
+         `mri_protocol` table or is excluded by the optional additional file 
+         checks present in the `mri_protocol_checks` table, then the MINC 
+         image of this scan will be stored into the 
+         `mri_protocol_violated_scans` table. This violated image is
```
Comment (MounaSafiHarab) [inline docs/02-Install.md]: I believe that if the protocol_checks return the excluded outcome, the logging is not done in mri_violated_scans but rather in mri_violations_log.

**Comment by cmadjar** [comment]
@MounaSafiHarab You are totally right. I always get confused with the different types of violations... Is this better now?

File: docs/02-Install.md [hunk 2]
```diff
@@ -93,13 +93,18 @@ Typically, images insertion into LORIS is performed via the following steps:
...
+         table and passes the optional additional file checks present in the 
+         `mri_protocol_checks` table, then the MINC image will be stored into 
+         the `files` tables. This inserted image is then accessible via the 
+         **Imaging Browser** module and can be displayed in 3D using 
+         **BrainBrowser**.
 
     b. If a scan does not match any of the protocol defined in the
          `mri_protocol` table, then the MINC image of this scan will be stored
-         into the `mri_protocol_violated_scans` table. This violated image is
+         into the `mri_protocol_violated_scans` table. Additionally, scans that
```
Comment (MounaSafiHarab) [inline docs/02-Install.md]: "in" instead of "into"

File: docs/02-Install.md [hunk 3]
```diff
@@ -93,13 +93,18 @@ Typically, images insertion into LORIS is performed via the following steps:
...
+         were excluded by the optional additional file checks present in the
+         `mri_protocol_checks` table will be inserted into the
+         `mri_violations_log` table. Both types of violated scans are
```
Comment (MounaSafiHarab) [inline docs/02-Install.md]: How about rewording like this:

"... in the `mri_protocol_violated_scans` table. Scans that have an identified protocol but that do not meet the **optional** criteria defined in the `mri_protocol_checks` table will be logged into the `mri_violations_log` table. ..."

**Comment by cmadjar** [comment]
@MounaSafiHarab thank you! I integrated your comments.
